.github/workflows/prepare-binary.yml

```diff
@@ -32,13 +32,15 @@ jobs:
         run: |
           mkdir -p binaries
           cp target/production/tanssi-node binaries/tanssi-node
+          cp target/production/tanssi-relay binaries/tanssi-relay
           cp target/production/container-chain-frontier-node binaries/container-chain-frontier-node
           cp target/production/container-chain-simple-node binaries/container-chain-simple-node
       - name: Save parachain custom binary
         if: ${{ matrix.cpu != '' }}
         run: |
           mkdir -p binaries
           cp target/production/tanssi-node binaries/tanssi-node-${{matrix.cpu}}
+          cp target/production/tanssi-relay binaries/tanssi-relay-${{matrix.cpu}}
```

Comment:
I think the best would be to put this into a separate job (I think that is what we agreed yesterday). Something like prepare-tanssi-relay-binary.yml or prepare-starlight-binary.yml

Reply:
Sure thing onto it now!

Reply:
Updated here https://github.com/moondance-labs/tanssi/pull/696/commits/c5f31b35858312bf5ea3eda86b63def77f53400b